keps/sig-storage/2644-honor-pv-reclaim-policy/README.md

```diff
@@ -286,6 +326,36 @@ Will pick the new behavior as described before.
 
 There will be unit tests for the feature `HonorPVReclaimPolicy` enablement/disablement.
 
+### Monitoring Requirements
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can an operator determine if the feature is in use by workloads?**
+
+The presence of finalizer described above in Persistent Volume will indicate that the feature is enabled.
+The finalizer `external-provisioner.volume.kubernetes.io/finalizer` will be present on the CSI Driver volumes.
+The finalizer `kubernetes.io/pv-controller` will be present on In-Tree Plugin volumes.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine
+  the health of the service?**
+  - [X] Metrics
+    - Metric name: For CSI Driver volumes, `persistentvolume_delete_duration_seconds` metric is used to track the time
+      taken for PV deletion. `volume_operation_total_seconds` metrics tracks the end-to-end latency for delete
+      operation grouped by plugin name.
```

Comment:
Is `volume_operation_total_seconds` for both CSI driver volumes and in-tree plugin volumes?

Reply:
both, I've clarified it.